### DISCUSSION TITLE
Typing mistake in compose.yaml in readme

### DISCUSSION THREAD

**Discussion by Chucky2401** [garethgeorge/backrest#466] [General]
Hi,
There is a typing mistake in the compose.yaml in the readme.

For the 'config' folder volume, there is an extra 'q' for the container side:
```yaml
    volumes:
      - ./backrest/config:/configq
```

I made the mistake to copy/paste without checking, and when I updated to 1.5.0, I saw this error. Unfortunately, even if I copied the files from inside the container to the host, I didn't check the file, and I lost all my config 😅.

Anyway, don't worry, I just started to use Backrest not a big loss.
But for future users, don't make the same mistake.

**Comment by garethgeorge** [comment]
Thanks for the notice on this, sorry you ran into trouble here! That’s a pretty small but really quite bad typo on my part. Just went ahead and updated the README.md.
